packages/components/src/components/form/Select/index.tsx

```diff
@@ -11,7 +11,7 @@ import ReactSelect, {
     SelectInstance,
```

Comment:
Probably not related to changes in this component, but it is select
<img width="537" alt="Screenshot 2022-09-09 at 10 19 10" src="https://user-images.githubusercontent.com/33235762/189305601-bedf8127-885c-4960-845d-5b08b8e41597.png">

Reply:
266e3024f0bfac65f1a20cf0be353499d9d8642e